src/style/global.scss

```diff
@@ -55,3 +55,14 @@ html {
     }
   }
 }
+
+@media (prefers-reduced-motion: reduce) {
+  *:not(.iui-fill),
+  *::before,
+  *::after {
```

Comment:
1. Why is `.iui-fill` excluded?
2. Applying to `*` will impact user styles.
3. I think we could do `[class*='iui-']` instead, but then it won't work for child elements of our components.
4. If we do `[class*='iui-'], [class*='iui-'] *`, then it will still impact user styles (`*` will target everything under `.iui-body`).

I think we might need to move this to a mixin, which we can include in our components. Or even inside `iui-reset`.

Reply:
`.iui-fill` is the animated part within a progress indicator.  I now realize that the `.iui-fill` is only applied on the radials, the linears had no class name and the CSS was targeting `> *`.  Next push will change that so the linear also use `.iui-fill`.